packages/react/src/components/form/Checkbox/Group/Group.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ChangeEventHandler, ReactNode } from 'react';
+import React, { forwardRef, createContext } from 'react';
+import cn from 'classnames';
+
+import type { FieldsetProps } from '../../Fieldset';
+import { Fieldset } from '../../Fieldset';
+import type { CheckboxProps } from '../Checkbox';
+
+import classes from './Group.module.css';
+
+export type CheckboxGroupContextProps = {
+  value?: string | ReadonlyArray<string> | number;
+  defaultValue?: string | ReadonlyArray<string> | number;
+  required?: boolean;
+} & Pick<CheckboxProps, 'onChange'>;
+
+export const CheckboxGroupContext =
+  createContext<CheckboxGroupContextProps | null>(null);
+
+export type CheckboxGroupProps = {
+  /** Collection of `Checkbox` components */
+  children?: ReactNode;
+  /** Controlled state for `Checkbox` */
+  value?: string | ReadonlyArray<string> | number;
+  /** Default checked `Checkbox` */
+  defaultValue?: string | ReadonlyArray<string> | number;
+  /** Callback event with changed `Checkbox` */
+  onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
Denne returnerer bare `onChange`-eventet til boksen man trykker på. Som utvikler ville jeg forventet å få en liste med alle valgte verdier, f.eks. ['vanilje', 'sjokolade'] hvis man har valgt vanilje og sjokolade, ev. at det fantes en annen prop hvor man kunne få denne informasjonen.

Reply:
Ja, tenkte kanskje vi kunne ha en egen prop for listen av valgte values. `toggledValues` eller `onToggleValues` noe slikt? Har heller ikke oppdatert typene her enda da det skal jo være en liste med `value`

Reply:
Endret `onChange` til å returnere liste med valgte verdier siden dette er tross alt en group komponent dedikert til sjekkbokser.

Tok og en titt på hva andre komponentbibliotek der ute og alle følgte samme tilnærming. 

Har og gjort endring samme endring i Radio #691 slik at forventet oppførsel er lik